examples/ct/ctmapper/README.md

```diff
@@ -25,20 +25,24 @@ go build ./examples/ct/ctmapper/lookup
 go build ./cmd/createtree/
 tree_id=$(./createtree \
```

Comment:
IIRC `./createtree` returns a whole json structure.  In KT, we used [jq](https://stedolan.github.io/jq/) to parse the output and obtain the `tree_id`.  Is something similar needed here?

Reply:
No, it doesn't - I just tried it and the output was simply the tree ID:

$ ./createtree  --admin_server=localhost:8090 --tree_type=MAP --hash_strategy=TEST_MAP_HASHER --signature_algorithm=ECDSA
9216245476404281793

Reply:
Maybe it used to output JSON?

Reply:
It prints only the tree ID. It's explicitly meant to be used this way.

https://github.com/google/trillian/blob/master/cmd/createtree/main.go#L21
https://github.com/google/trillian/blob/master/cmd/createtree/main.go#L179.